src/FormattingSettingsService.ts

```diff
@@ -159,24 +166,35 @@ export class FormattingSettingsService implements IFormattingSettingsService {
         return formattingModel;
     }
 
-    private buildFormattingSlices({slices, objectName, sliceNames, formattingSlices }: IBuildFormattingSlicesParams) {
+    private buildFormattingSlices({ slices, objectName, sliceNames, selectorsMap, formattingSlices }: IBuildFormattingSlicesParams) {
```

Comment:
Create selector map here inside the method, You dont need to pass it from another method as a parameter